### PR TITLE
Change order of possible-owner organizations to alphabetical

### PR DESCRIPTION
**Pull request by eneuschild**
Hello again! :)

This is an attempt to address Issue #13090 . 
Any feedback is greatly appreciated; I'm happy to learn.

### CODE REVIEW

File: models/org.go [hunk 1]
```diff
@@ -476,7 +476,8 @@ func GetOrgsCanCreateRepoByUserID(userID int64) ([]*User, error) {
 		Join("INNER", "`team`", "`team`.id = `team_user`.team_id").
 		Where(builder.Eq{"`team_user`.uid": userID}).
 		And(builder.Eq{"`team`.authorize": AccessModeOwner}.Or(builder.Eq{"`team`.can_create_org_repo": true})))).
-		Desc("`user`.updated_unix").Find(&orgs)
+		Asc("`user`.name").
```
Comment (zeripath) [inline models/org.go:479]: Would lower name be better? Or should we have the capitalised organizations before the non-capitalised ones?

```suggestion
		Asc("`user`.lower_name").
```

Comment (lafriks) [reply models/org.go:479]: Database will sort using natural language rules so it does not matter much

**Review by zeripath — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lunny** [comment]
Why this is better than before? We may need options but not change it.

**Comment by lafriks** [comment]
@lunny in combobox it does not makes much sense to sort by last updated time imho

**Pull request merged**